models/order.js

```diff
@@ -0,0 +1,42 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Order extends Model {}
+
+Order.init(
+  {
+
+    order_id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    customer_id: {

```

Comment:
customer ID is the foreign key?
if it is do we need to use references: https://sequelize.org/docs/v6/core-concepts/model-basics/